keyboards/svalboard/keymaps/vial/vial.json

```diff
@@ -45,6 +45,26 @@
         "shortName": "Mouse\nKey\nTimer",
         "title": "Toggle between various settings of the mouse keys timer.",
         "name": "SV_MH_CHANGE_TIMEOUTS"
+      },
+      {
+        "shortName": "Caps\nWord\nToggle",
+        "title": "Toggle Caps Word.",
+        "name": "SV_CAPS_WORD"
```

Comment:
Does this toggle Caps Word as a replacement to Capslock, or just activate Caps Word itself, since it should turn off after operating?

Reply:
https://docs.qmk.fm/features/caps_word